x-pack/legacy/plugins/infra/public/pages/metrics/layouts/aws.ts

```diff
@@ -0,0 +1,247 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { i18n } from '@kbn/i18n';
+import { InfraMetric } from '../../../graphql/types';
+import { InfraFormatterType } from '../../../lib/lib';
+import {
+  InfraMetricLayoutCreator,
+  InfraMetricLayoutSectionType,
+  InfraMetricLayoutVisualizationType,
+} from './types';
+
+export const awsLayoutCreator: InfraMetricLayoutCreator = theme => [
```

Comment:
If we change the function signature to this:
```
export const awsLayoutCreator = (theme: EuiTheme): InfraMetricLayout[] => [
...
];
```
then Typescript will catch errors in the `InfraMetricLayout` schema (like the `requires` field below). I didn't realized this until I created the screen cast for adding an additional section to these layouts that the typing system was broken for these files. I've fixed the other layouts in https://github.com/elastic/kibana/pull/42710